aclk/aclk_otp.c

```diff
@@ -275,6 +275,12 @@ static int parse_passwd_response(const char *json_str, struct auth_data *auth) {
     return rc;
 }
 
+#define JSON_KEY_ERTRY   "errorNonRetryable"
+#define JSON_KEY_EDELAY  "errorRetryDelaySeconds"
+#define JSON_KEY_EEC     "errorCode"
+#define JSON_KEY_EMSGKEY "errorMsgKey"
+#define JSON_KEY_EMSG    "errorMessage"
+#if JSON_C_MINOR_VERSION >= 13
```

Comment:
Can't we use the `#else` path for all versions? Did they broke compatibility or did they mark functions as deprecated?

Reply:
they are new functions present from version 13 only

Reply:
Yeah you can use #else for new JSON-C as well.... I implemented it with new fnc originally and only then found out it is failing CI on older systems. I prefer to keep the new one as it is simpler and remove #else when all our systems update JSON-C

Reply:
I don't like the idea of maintaining two different versions of the same functionality, but that's just my opinion so it's fine for this PR.

Any reason for not setting `aclk_block_until` and `aclk_disable_runtime` in the `#else` clause?

Reply:
yep, I understand your point... I wish new JSON_C is in all systems soon (or the old systems will go into unsupported status)

fixed the else clause bug